### ISSUE TITLE
Albums don't show more than 100 tracks

### ISSUE THREAD

**Issue by Carter0108** [fast4x/RiMusic#1824]
### Steps to reproduce the bug

Go to an album with more than 100 tracks.

### Expected behavior

Show all songs on the album.

### Actual behavior

Only shows the first 100.

### Screenshots/Screen recordings

![Screenshot_20240510-231146](https://github.com/fast4x/RiMusic/assets/55948138/efd59a01-06df-489c-90e9-925d4c7d4f6e)
![Screenshot_20240510-231201](https://github.com/fast4x/RiMusic/assets/55948138/6bec3e60-2517-45cb-a92f-6e520777e4d5)

### RiMusic version

0.6.34

### Android version

Android 14

**Comment by dachshund7** [comment]
I'm running into this issue with a box set of Handel's music. Here is a link to one of the albums: https://music.youtube.com/playlist?list=OLAK5uy_ma2DmhtU7zLegQx9aussci7Mf2dZs_hIM

There are 152 tracks, but the app only plays the first 100.

**Comment by twistios** [comment]
This is a bit larger problem. At least there now is a solution for the case that an artist has more than 100 songs, because this also was not working previously and there are more artists with >100 songs compared to the albums with >100 songs.

**Comment by DatHypnoboi** [comment]
Really hope this can be fixed

**Comment by dachshund7** [comment]
I've been using this Python script as a workaround that uses the [ytmusicapi](https://github.com/sigma67/ytmusicapi) library to create a CSV file of an album that can then be imported into RiMusic as a playlist.

```python
import argparse
import csv
import sys

from ytmusicapi import YTMusic


def write_csv(playlist_id: str) -> None:
    ytmusic = YTMusic()
    browse_id = ytmusic.get_album_browse_id(playlist_id)
    album = ytmusic.get_album(browse_id)
    playlist = ytmusic.get_watch_playlist(playlistId=playlist_id, limit=album["trackCount"])

    fieldnames = [
        "PlaylistBrowseId",
        "PlaylistName",
        "MediaId",
        "Title",
        "Artists",
        "Duration",
        "ThumbnailUrl",
        "AlbumId",
        "AlbumTitle",
        "ArtistIds",
    ]

    writer = csv.DictWriter(sys.stdout, fieldnames=fieldnames)
    writer.writeheader()

    for track in playlist["tracks"]:
        writer.writerow({
            "PlaylistBrowseId": "",
            "PlaylistName": album["title"],
            "MediaId": track["videoId"],
            "Title": track["title"],
            "Artists": "",
            "Duration": track["length"],
            "ThumbnailUrl": album["thumbnails"][-1]["url"],
            "AlbumId": browse_id,
            "AlbumTitle": album["title"],
            "ArtistIds": ",".join(a["id"] for a in track["artists"] if a["id"]),
        })


def main() -> None:
    parser = argparse.ArgumentParser()
    parser.add_argument("playlist_id", help="Playlist ID for YouTube Music album")
    args = parser.parse_args()
    write_csv(args.playlist_id)


if __name__ == "__main__":
    main()
```

Once you have the Playlist ID of the album, you can run the code like this:
```bash
$ python main.py OLAK5uy_lR1Bq94o4E8okd3G4AyajcbAMHyErxgXI > persona5.csv
```

**Comment by DatHypnoboi** [comment]
Not sure what the issue is, but whenever I try to import a playlist generated with your script, it just crashes my RiMusic lol

**Comment by dachshund7** [comment]
Are you using the latest version of RiMusic? The format for the CSV files got changed recently and some new columns were added.

**Comment by DatHypnoboi** [comment]
Yeah, auto-updated a few hours ago, do the script generated csv's not work on the new version?

**Comment by dachshund7** [comment]
Oh, I didn't see the new update today. I use the Accrescent app and the update hasn't appeared on there yet. I'll take another look.

**Comment by dachshund7** [comment]
I did a fresh install of v0.6.71 on my tablet, and I was able to import a file, so I am not sure what's going wrong.

**Comment by DatHypnoboi** [comment]
I guess I'll try a fresh install as well, haven't been using it that long so not too much to carry over

**Comment by dachshund7** [comment]
Just to be sure, are you trying to import the file by using this icon here?

![Image](https://github.com/user-attachments/assets/d44155bb-060f-463d-87c9-d38818ce622d)

**Comment by DatHypnoboi** [comment]
Yep

**Comment by DatHypnoboi** [comment]
Still not working on a fresh install, guess I'll just wait till the app gets fixed ::|

**Comment by dachshund7** [comment]
@fast4x Will this be fixed in a new release?

**Comment by dachshund7** [comment]
I guess not...

**Comment by fast4x** [comment]
For now isn't planned new versions...
